.github/workflows/cypress.yaml

```
@@ -81,58 +77,23 @@
-          parallel: true
-          group: "UI-Chrome"
-        env:
-          CYPRESS_SPLIT_TESTS: "true"
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NODE_OPTIONS: --max_old_space_size=4096
-          COMMIT_INFO_MESSAGE: ${{github.event.pull_request.title}}
-          COMMIT_INFO_SHA: ${{github.event.pull_request.head.sha}}
-          SPLIT: ${{ strategy.job-total }}
-          SPLIT_INDEX: ${{ strategy.job-index }}
+      - name: Fix Permissions for Cypress Test Files
+        run: chmod -R +rwx cypress/e2e  
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)